doc/README.md

```diff
@@ -404,6 +406,67 @@ click_info {
 
 ***
 
+
+## <a name="py3"></a>Py3 module helper
+
+Py3 is a special helper object that gets injected into
+py3status modules, providing extra functionality.  A module can access it via
+`self.py3` in its `Py3status` class
```

Comment:
→ A module can access it via the `self.py3` instance attribute of its `Py3status` class.